dev-docs/bidders/indexExchangeLegacy.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Index Exchange Legacy
+description: Prebid Index Exchange Legacy Bidder Adaptor
+
+top_nav_section: dev_docs
+nav_section: reference
+
+hide: true
+biddercode: indexExchange
+
+biddercode_longer_than_12: true
+media_types: video
+
+---
+
+## Legacy Adapter
+
+This is the IndexExchange legacy adapter and you should reference the [new IX adapter for bidder information]({{site.baseurl}}/dev-docs/bidders.html#ix).
```

Comment:
Better to link to the same area: /dev-docs/bidders/indexExchange.html ?

Reply:
whoops - good catch thanks.